packages/kokoas-client/src/pages/projInvoice/hooks/useResolveParams.ts

```diff
@@ -50,6 +50,7 @@ export const useResolveParams = () => {
         draft.custName = custData.custNames.value;
         newEstimates?.forEach((data, idx) => {
           const tgtBilledAmount = datInvoicesTotal?.find(({ dataId }) => dataId === data.dataId)?.billedAmount ?? 0;
+          const tgtCreatedAmount = datInvoicesTotal?.find(({ dataId }) => dataId === data.dataId)?.createdAmount ?? 0;
```

Comment:
```suggestion
          const targetInvoice = datInvoicesTotal?.find(invoice => invoice.dataId === data.dataId);
          const {
            billedAmount = 0,
            createdAmount = 0,
          } = targetInvoice || {};
```

datInvoicesTotalにおけるfindの繰り返し利用を避け、配列から目的の値を直接取得する方法を使用することができます。

Reply:
ご指摘ありがとうございます。
上記内容で修正しました。
[0b5bb97](https://github.com/Lorenzras/yumecoco-monorepo/pull/147/commits/0b5bb97efdd8ccbceaf92b91ab69bb1749313896)にて、ご確認をお願いいたします。